tests/test_common_gpu.py

```diff
@@ -826,11 +826,7 @@ def test_8bit_lora_mixed_adapter_batches_lora(self):
         # check that we can pass mixed adapter names to the model
         # note that with 8bit, we have quite a bit of imprecision, therefore we use softmax and higher tolerances
         torch.manual_seed(3000)
-        bnb_config = BitsAndBytesConfig(
-            load_in_8bit=True,
-            bnb_4bit_use_double_quant=False,
-            bnb_4bit_compute_dtype=torch.float32,
-        )
+        bnb_config = BitsAndBytesConfig(quantization_config=BitsAndBytesConfig(load_in_8bit=True))
```

Comment:
```suggestion
        bnb_config = BitsAndBytesConfig(load_in_8bit=True)
```
Maybe a regex error 🤯

Reply:
Good catch, yeah, overeager search and replace.